NotebookAnalyzers/main.py

```diff
@@ -0,0 +1,22 @@
+from fastapi import FastAPI
+import pynblint
+import linters
+
+app = FastAPI()
+nb_linter = linters.NbLinter()
+repo_linter = linters.RepoLinter()
+
+
+@app.get('/')
+def index():
+    return {'data': {'message': 'Welcome to the pynblint API'}}
+
+
+@app.get('/linters')
+def linters_descr():
```

Comment:
Userei un identificatore più generico per questa funzione:

```suggestion
def get_linters_list():
```

Non sappiamo se in futuro se gli oggetti restituiti conterranno solo le descrizioni dei linter disponibili o anche altri metadati.